imports/plugins/included/paystack/lib/api/paystack.js

```diff
@@ -0,0 +1,24 @@
+import request from "request";
+
+export const Paystack = {};
+
+const paystackHeaders = (secret) => {

```

Comment:
Consider improving the documentation in your PR. Almost all funcitons do not have any form of documentation

Reply:
I will fix this right now